api/services/userService.js

```diff
@@ -12,17 +15,15 @@ const hashPassword = async (plainTextPassword) => {
 }
 
 const signUp = async (name, email, password, phoneNumber) => {
-    const EMAIL_REGEX    = /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/
-	const PASSWORD_REGEX = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{8,})/
-
-    if ( !EMAIL_REGEX.test(email)) {
+    
+    if (validator.emailValidator(email)) {
```

Comment:
해당 부분도 바로 함수로 적용하면 됩니다.
아래 signIn 함수도 마찬가지입니다!

```suggestion
    emailValidator(email);
    passwordValidator(password);
```